NetKAN/KerbalWind.netkan

```diff
@@ -0,0 +1,20 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "KerbalWind",
+    "$kref"        : "#/ckan/github/DaMichel/KerbalWind",
+    "name"         : "Kerbal Wind",
+    "abstract"     : "Adds wind to the game. It opens a dialog box where you can set wind direction and speed. Inspired by KerbalWeatherSystem by silverfox8124. But this is much simpler, omitting the actual weather simulation.",
+    "license"      : "MIT",
+    "ksp_version"  : "0.90",
+    "recommends" : [
+        { "name" : "Toolbar" }
+    ],
+	"requires" : [
```

Comment:
Hey @DaMichel , could you please replace `requires` with `depends` ? It won't work otherwise :)